src/app/zap-templates/zcl/data-model/chip/joint-fabric-administrator-cluster.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ICACResponseStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="InvalidPublicKey" value="0x01"/>
+    <item name="InvalidICAC" value="0x02"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Joint Fabric Administrator</name>
+    <code>0x0753</code>
+    <define>JOINT_FABRIC_ADMINISTRATOR_CLUSTER</define>
+    <description>Joint Fabric Administrator Cluster.</description>
+
+    <attribute side="server" code="0x0000" name="ADMINISTRATOR_FABRIC_INDEX" define="ADMINISTRATOR_FABRIC_INDEX" type="fabric_idx" default="0">
```

Comment:
Why is this field in upper case?

Reply:
default="0" means kUndefinedFabricIndex, which is an appropriate value when the cluster is enabled but doesn't want to participate in JF

Reply:
thx, @vijs , makes sense.